src/Kiota.Builder/WorkspaceManagement/ApiClientConfiguration.cs

```diff
@@ -15,6 +16,10 @@ public class ApiClientConfiguration : BaseApiConsumerConfiguration, ICloneable
     /// </summary>
     public string Language { get; set; } = string.Empty;
     /// <summary>
+    /// The type access modifier to use for the client types.
+    /// </summary>
+    public string TypeAccessModifier { get; set; } = "Public";
```

Comment:
@andrueastman, could we use the enum type here instead of a string

Reply:
I'm okay with this really. It would however be inconsistent with other enum value params like Language(line above) so we may need to change that too. 
I also suspect it may break the serialization of the configuration objects into lock/workspace files as the serializer would need to be configured to serialize enums as strings (by default I believe STJ will write this as a number)
What we could do is may sort this out separately so that it's aligned across the board?

Reply:
ping @calebkiage

Reply:
@NKnusperer I believe you will also need to run `dotnet format` to have the liniting checks on the PRpassing as well

Reply:
I agree. We can make that improvement in a different PR